text/0096-auto-batching.md

```diff
@@ -83,6 +60,6 @@ N/A
 
 - Extends it for all consecutive payload types.
 - Add a filter capability by `batchUid` on the `/tasks` endpoints.
-- Do not fail the entire transaction if a document is not valid. Report the documents that could not be indexed to the user.
-- Enable auto-batching by default.
-- Optimize some tasks sequence, for example if there is a document addition followed by an index deletion, we could skip the document addition
\ No newline at end of file
+- <s>Do not fail the entire transaction if a document is not valid. Report the documents that could not be indexed to the user.</s>
```

Comment:
Don't know if this format is ok for you @gmourier to say "it's done" without completely remove them

Reply:
The current behavior fails the entire task if one document is invalid. However, an entire batch doesn't fail anymore if a contained task failed.

Reply:
@curquiza This format might suggest that we have decided not to implement future possibilities. WDYT?